src/Package.js

```diff
@@ -60,12 +61,27 @@ export default class Package {
     return this._package.scripts || {};
   }
 
+  set versionSerializer(versionSerializer) {
+    const previousSerializer = this._versionSerializer;
+
+    this._versionSerializer = versionSerializer;
+
+    if (previousSerializer) {
```

Comment:
When does this ever happen? Just being defensive?

(It seems strange to assign the versionSerializer multiple times)

Reply:
Yes, it's just being defensive. Currently, it is not likely that second serializer would be used in one run. YAGNI and remove?

Reply:
Yes, please.